manim/cli/render/commands.py

```diff
@@ -136,4 +138,40 @@ def __repr__(self):
             except Exception:
                 console.print_exception()
 
+    if config.notify_outdated_version:
+        manim_info_url = "https://pypi.org/pypi/manim/json"
+        warn_prompt = "Cannot check if latest release of manim is installed"
+        req_info = {}
+
+        try:
+            req_info = requests.get(manim_info_url, timeout=3)
```

Comment:
don't set a timeout manually 
```suggestion
            req_info = requests.get(manim_info_url)
```